java/src/jmri/jmrit/logixng/tools/swing/CallModuleParameterTableModel.java

```diff
@@ -108,13 +108,17 @@ public boolean isCellEditable(int row, int col) {
     @Override
     public void setValueAt(Object value, int rowIndex, int columnIndex) {
         ParameterData variable = _parameterData.get(rowIndex);
-        
+
         switch (columnIndex) {
             case COLUMN_NAME:
                 variable._name = (String) value;
                 break;
             case COLUMN_INPUT_TYPE:
-                variable._initialValueType = (InitialValueType) value;
+                if (value != null) {
```

Comment:
This looks fine to me.  I don't see why it's providing a null value, but this is a fine way to deal with it.

Incidentally, down at line 244 you have a routine that is annotated for "unchecked" because you're casting to JComboBox.  I think it might work to check against and cast to JComboBox<?> in that case.

Reply:
> Incidentally, down at line 244 you have a routine that is annotated for "unchecked" because you're casting to JComboBox. I think it might work to check against and cast to JComboBox<?> in that case.

Thank you. I have fixed both cases.

Unfortunately it doesn't help. I still get the error:
```
1. ERROR in /home/runner/work/JMRI/JMRI/java/src/jmri/jmrit/logixng/tools/swing/CallModuleParameterTableModel.java (at line 247)
	(JComboBox<InitialValueType>) event.getSource();
	^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Type safety: Unchecked cast from Object to JComboBox<SymbolTable.InitialValueType>
```
So I need to keep the SuppressWarning.

Reply:
Sorry about that.  I should have tried it before suggesting it.

Reply:
> Sorry about that. I should have tried it before suggesting it.

No problem! I do the same sometimes. And even if it didn't work this time, it's always useful to be aware of the `<?>` syntax.